src/main/java/org/prebid/server/bidder/ix/IxAdapter.java

```diff
@@ -141,6 +126,29 @@ private static IxParams parseAndValidateParams(AdUnitBid adUnitBid) {
         return params;
     }
 
+    private static boolean isValidIxSize(Format format, List<Integer> sizes) {
+        return format.getW().equals(sizes.get(0)) && format.getH().equals(sizes.get(1));
```

Comment:
1. Do we sure `format.getW()` & `format.getH()` are not NULLs?
2. Do we sure `sizes` has size of 2 items ?

Pls, take a look at `Objects.equals(...)` helper method.